.github/ISSUE_TEMPLATE/extension-request.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Extension request
 about: Missing an extension?
-title: 'Insert name of the extension here'
-labels: '🙋  Extension/feature request'
+title: Insert name of the extension here
+labels: "\U0001F64B  Extension/feature request"
```

Comment:
Should not need to be changed, same as my other comment. Not sure though, if you feel like it should be this way, we can keep that.